src/machine_charm.py

```diff
@@ -154,13 +154,15 @@ def _additional_log_configs(self) -> List[Dict[str, Any]]:
                 "scrape_configs": [
                     {
                         "job_name": "varlog",
-                        "static_configs": {
-                            "targets": ["localhost"],
-                            "labels": {
-                                "__path__": "/var/log/*log",
-                                **self._principal_labels,
-                            },
-                        },
+                        "static_configs": [
+                            {
+                                "targets": ["localhost"],
+                                "labels": {
+                                    "__path__": "/var/log/*log",
```

Comment:
Do we need a period there?
```suggestion
                                    "__path__": "/var/log/*.log",
```

Reply:
Syslog.. scratch that :)